pkgs/applications/audio/termusic/default.nix

```diff
@@ -1,31 +1,38 @@
 { lib
 , stdenv
 , rustPlatform
-, fetchCrate
-, fetchpatch
+, fetchFromGitHub
+, protobuf
+, dbus
+, openssl
 , pkg-config
 , alsa-lib
 , darwin
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "termusic";
-  version = "0.7.10";
+  version = "v0.7.11";
```

Comment:
```nix
version = "0.7.11";
```

Reply:
?

Reply:
The `v` is part of the GitHub tag and shouldn't end up in the `version` attribute itself...

Reply:
Ok got it. I fully agree.

Reply:
Thanks for this hint, I actually disliked adding that 'v' there, but didn't see the possibility to expand the variable inside the string.